mitodl/mitxpro
# B2b Bulk Course/Program dates

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1932 

#### What's this PR do?
This PR fixes some bugs associated with products on B2B bulk [page](https://xpro.mit.edu/ecommerce/bulk/). e.g. Available courses/programs dropdown values and Available course/program runs dates dropdown values.

#### How should this be manually tested?
Go to [B2B Bulk page](https://xpro.mit.edu/ecommerce/bulk/) and select course/program dropdown value. Depending upon the selection the courses/programs names dropdown should be populated as per cases discussed in [this issue](https://github.com/mitodl/mitxpro/issues/1932)
